WindowsServerDocs/administration/windows-commands/mount.md

```diff
@@ -38,7 +38,7 @@ mount [-o <option>[...]] [-u:<username>] [-p:{<password> | *}] {\\<computername>
 | -o casesensitive | Forces file lookups on the server to be case sensitive. |
 | -o fileaccess=`<mode>` | Specifies the default permission mode of new files created on the NFS share. Specify *mode* as a three-digit number in the form *ogw*, where *o*, *g*, and *w* are each a digit representing the access granted the file's owner, group, and the world, respectively. The digits must be in the range 0-7, including:<ul><li>**0:** No access</li><li>**1:** x (execute access)</li><li>**2:** w (write access)</li><li>**3:** wx (write and execute access)</li><li>**4:** r (read access)</li><li>**5:** rx (read and execute access)</li><li>**6:** rw (read and write access)</li><li>**7:** rwx (read, write, and execute access)</li></ul> |
 | -o lang=`{euc-jp|euc-tw|euc-kr|shift-jis|Big5|Ksc5601|Gb2312-80|Ansi)` | Specifies the language encoding to configure on an NFS share. You can use only one language on the share. This value can include any of the following values:<ul><li>**euc-jp:** Japanese</li><li>**euc-tw:** Chinese</li><li>**euc-kr:** Korean</li><li>**shift-jis:** Japanese</li><li>**Big5:** Chinese</li><li>**Ksc5601:** Korean</li><li>**Gb2312-80:** Simplified Chinese</li><li>**Ansi:** ANSI-encoded</li></ul> |
-| -u:`<username>` | Specifies the user name to use for mounting the share. If *username* isn't preceded by a backslash (**\**), it's treated as a UNIX user name. |
+| -u:`<username>` | Specifies the user name to use for mounting the share. If *username* isn't preceded by a backslash (**\\**), it's treated as a UNIX user name. |
```

Comment:
Looks good so far. Since we are already editing this line, I think we should add spaces before & after the backslash to make it more visible ( within the parentheses ) :
```suggestion
| -u:`<username>` | Specifies the user name to use for mounting the share. If *username* isn't preceded by a backslash ( **\\** ), it's treated as a UNIX user name. |
```

Reply:
Sure thing. On the same page there's also instances where it shows an example of an asterisk, maybe it would be good to add spaces there too?

Reply:
Though perhaps out of the scope of this PR.

Reply:
Fair point, although to me it seems like less of a problem with the asterisks, because they seem to have some minimal whitespace around them anyway. The backslash seems more problematic, because it joins the parentheses like a ligature, making it look like (\\) is a single symbol or character.

Reply:
Ah I see, that's alright then, thanks!